### DISCUSSION TITLE
Tablet operation without keyboard

### DISCUSSION THREAD

**Discussion by DavidSpeckMD** [sumatrapdfreader/sumatrapdf#4052] [Documentation]
I would like to use Sumatra to display sheet music PDF files on my organ music stand.  The tablet has no keyboard showing in normal operation.  Is there a setting to make it display the pages sequentially left to right, instead of stacked vertically, and is there a way to make it scroll one complete page at a time with a swipe left.  Thanks,  Dave

**Comment by GitHubRulesOK** [comment]
For horizontal pages you can spin 90 degrees but then trickier to manually slide unless the touch is following the rotation, 
Many musicians invest in a pedal key/mouse (and commands can be assigned to keycode) but cost for best value a challenge, unless you can test variations for free.

**Reply by DavidSpeckMD** [reply]
Thanks for the response.  I should have specified that I use the tablet in portrait orientation, as most of my music  is printed that way.  On the organ, both hands a feet get going at the same time, so hanging a foot pedal on the tablet is not my preferred option.  

Is there a parameter to make Sumatra scroll an integer page at at a time?

**Reply by GitHubRulesOK** [reply]
in continuous mode right arrow icon will jump to top of next page 
not much help if your piece is always low on a page
but in maximised single page mode the right arrow icon should jump a page downwards.

However on tablets I have to increase toolbar size to be able to stab it quickly, without going back however try `F5` where  page tap anywhere should move forwards BEWARE to go back needs right click so was long hold but not tested in recent ?
